cwms_radar_api/src/main/java/cwms/radar/data/dao/RatingSetDao.java

```diff
@@ -92,30 +118,29 @@ public void delete(String officeId, String ratingSpecId) throws IOException, Rat
 	{
 		try
 		{
-			dsl.connection(c ->
-			{
-//				deleteWithRatingSet(c, officeId, ratingSpecId);
+			dsl.connection(c -> {
+				//				deleteWithRatingSet(c, officeId, ratingSpecId);
```

Comment:
This comment got further indented with the formatting change.
Is there a reason this commented out code even needs to remain?

Reply:
Nope. Will delete.